### DISCUSSION TITLE
Installing arcade in Python 3.12.6

### DISCUSSION THREAD

**Discussion by jvsk-py** [practical-tutorials/project-based-learning#628] [Q&A]
Hi there! I want to learn game development, and I navigated my way to the arcade library.

Naturally, I wanted to install the library, so I opened up a VENV in cmd (windows 11 without admin privileges) and typed in "python -m pip install arcade" and it did a bunch of stuff with a few errors saying that it isn't a problem with PIP and returned to the command line.

I also tried "pip3 install arcade" and did everything in the article: https://api.arcade.academy/en/2.5.7/installation_windows.html

This is also my first time posting on GitHub, so please don't refrain from calling me out on some mistakes and suggesting some things.

Thanks!

**Comment by ghost** [comment]
hello! can you share the error message? I recently had an error installing arcade but it was really just asking me to brew uninstall pillow first so that it can reinstall pillow in the arcade install.

**Reply by jvsk-py** [reply]
These are the messages, and frankly, I understand none of them. 😅
![Screenshot 2024-09-15 204606](https://github.com/user-attachments/assets/be65d86f-a10b-488a-97f0-f4358c5700ec)
![Screenshot 2024-09-15 204615](https://github.com/user-attachments/assets/07a25495-c160-4ef4-aaa1-bca5083dea80)

**Reply by ghost** [reply]
are you using pip3 or pipenv? pip3 install arcade or python3 -m pipenv install arcade should work for command line install but also to note you need to switch from python 3.12 to python 3.10 or 3.11 as arcade 2.6.17 isn't made for 3.12 yet as per the error.

**Reply by jvsk-py** [reply]
I am doing the following command:
python -m pip install arcade
I have tried it a few times.

Also, I do not know how to change Python version in VSC (I have already installed Arcade without a VENV)
